### DISCUSSION TITLE
Lost all shortcuts

### DISCUSSION THREAD

**Discussion by rolomoto** [Stellarium/stellarium#1948] [Q&A]
I installed 21.2 (thank you for the update) and shortcuts were working. I opened it today and they have all been reset to the defaults. Is there any way to recover them?
Thanks

**Comment by alex-w** [comment]
I guess you’re reseted configuration file when installing Stellarium and in this case all old parameters, include custom shortcuts, was lost. So, the answer is: no, recovery is not possible.
